ts/client/src/accounts/mangoAccount.ts

```diff
@@ -1973,6 +1959,62 @@ export class TokenConditionalSwap {
     return this.pricePremiumRate * 100;
   }
 
+  getCurrentlySuggestedPremium(group: Group): number {
+    const buyBank = this.getBuyToken(group);
+    const sellBank = this.getSellToken(group);
+    return TokenConditionalSwap.computePremium(
+      group,
+      buyBank,
+      sellBank,
+      this.maxBuy,
+      this.maxSell,
+      this.getMaxBuyUi(group),
+      this.getMaxSellUi(group),
```

Comment:
Why pass maxBuy and maxBuyUi if the bank is also passed? 

For the currently suggested premium, it could make sense to use the remaining buy/sell amounts, not the original maximums. Maybe it's half-filled already.

Reply:
> Why pass maxBuy and maxBuyUi if the bank is also passed?
Fixed

Reply:
Fixed both